app/js/constants/situationsFamiliales.js

```diff
@@ -13,7 +13,7 @@ angular.module('ddsCommon').constant('situationsFamiliales', [
     },
     {
         value: 'relation_libre',
```

Comment:
Unless this has important side-effects, please also update identifiers to avoid domain mismatch.